actions/insights/advisor_actions.py

```diff
@@ -45,7 +45,7 @@
 advisor_system = FuzzySlotMatch(
     "insights_advisor_system_kind",
     [
-        FuzzySlotMatchOption("rhel", ["rhel", "redhat", "red-hat", "red hat"]),
+        FuzzySlotMatchOption("rhel", ["rhel", "enterprise", "linux"]),
```

Comment:
This makes sense! It addresses a bug in https://github.com/RedHatInsights/astro-virtual-assistant/pull/367/files